plugins/pencilblue/templates/admin/content/articles/article_form.html

```diff
@@ -7,12 +7,26 @@
 		<fieldset>
 			<div class="tab-content">
 				<div class="tab-pane active" id="content">
+					<div class="form-group" ng-class="{'has-error': !isFieldValid(articleForm.headline)}">
```

Comment:
I'm fine with that, but it'll need to change on the page form too, for consistency.